packages/services/src/kernel/default.ts

```diff
@@ -225,6 +240,13 @@ export class KernelConnection implements Kernel.IKernelConnection {
     return this._specPromise;
   }
 
+  /**
+   * Check if this kernel supports JEP 91 kernel subshells.
+   */
+  supportsSubshells(): boolean {
```

Comment:
Why is this a method rather than a getter?

```suggestion
  get supportsSubshells(): boolean {
```

Reply:
No reason. It could be either, I picked one, I didn't know what is preferred in this project. I've changed it to a getter as requested.